src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -759,7 +759,7 @@ public IActionResult SendTransaction([FromBody] SendTransactionRequest request)
                     bool isUnspendable = output.ScriptPubKey.IsUnspendable;
                     model.Outputs.Add(new TransactionOutputModel
                     {
-                        Address = isUnspendable ? null : output.ScriptPubKey.GetDestinationAddress(this.network).ToString(),
+                        Address = isUnspendable ? null : output.ScriptPubKey.GetDestinationAddress(this.network)?.ToString(),
```

Comment:
In which cases will this the address be null?

Reply:
Cold staking scripts, being associated with two public keys, do not have a single deterministic address that can be derived from the keys alone. `GetDestinationAddress` would return null.